File: greeting/python/greeting_client.py
import grpc
from greeting_pb2 import GreetingRequest
from greeting_pb2_grpc import GreeterStub

channel = grpc.insecure_channel("localhost:50051")
client = GreeterStub(channel)

request = GreetingRequest(
    name="Andrew"
)

greeting_message = client.sayHello(request)

print(greeting_message)

File: greeting/javascript/greeting_client.js
const grpc = require('@grpc/grpc-js');
const protoLoader = require('@grpc/proto-loader');

const packageDefinition = protoLoader.loadSync('../../protobufs/greeting.proto', {});
const greetingPackage = grpc.loadPackageDefinition(packageDefinition);


const client = new greetingPackage.Greeter(
  // gRPC server to connect to
  'localhost:50051',

  // Use insecure mode
  grpc.credentials.createInsecure()
);

// Call the `sayHello` RPC method on the server
// {'name': 'Jason Bourne'} is the request body
client.sayHello({'name': 'Jason Bourne'}, (err, response) => {
  if (err) {
    console.log(err);
  } else {
    console.log(`From server`, JSON.stringify(response));
  }
});
